app/managers.py

```diff
@@ -0,0 +1,44 @@
+import sqlite3
+
+from models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connection = sqlite3.connect(
+            "D:/mate/py-actor-manager/cinema.sqlite"
```

Comment:
Removing the full path makes the code more portable and adaptable to different environments